src/main/resources/static/js/index.js

```diff
@@ -76,7 +77,7 @@ programInput.addEventListener("change",(e) => {
 
 
 buttonAdd.addEventListener("click", () => {
-    const exercise = Exercise.createExercise(exerciseInput.value, weightInput.value, repsInput.value, setsInput.value)
+    const exercise = Exercise.createExercise(exerciseInput.value, weightInput.value, repsInput.value, setsInput.value, notesInput.value)
```

Comment:
Not rejecting the PR over this, but usually if a constructor like this has so many arguments it runs off screen, then it should be formatted more like:

constructorThing(
    field1,
    field2,
    field3,
    field4
)